config/cops.yml

```diff
@@ -1,24 +1,25 @@
 Metrics/AbcSize:
   violation_points: 100_000
 Metrics/BlockNesting:
-  base_points: 30_000
+  base_points: 100_000
 Metrics/ClassLength:
-  base_points: 400_000
+  base_points: 800_000
+  violation_points: 75_000
 Metrics/CyclomaticComplexity:
-  base_points: 75_000
-  violation_points: 10_000
+  base_points: 1_000_000
```

Comment:
Does this effectively mean that any instance of Cyclomatic complexity makes a file a "B"?